libs/interpret-vision/src/lib/VisionExplanationDashboard/Controls/Flyout.tsx

```diff
@@ -111,175 +119,177 @@ export class Flyout extends React.Component<IFlyoutProps, IFlyoutState> {
+                  <Stack.Item className={classNames.imageContainer}>
+                    <Image
+                      src={`data:image/jpg;base64,${item?.image}`}
+                      className={classNames.image}
+                      imageFit={ImageFit.contain}
+                    />
+                  </Stack.Item>
+                </Stack>
+              </Stack.Item>
               <Stack.Item>
-                <Text variant="large" className={classNames.title}>
-                  {localization.InterpretVision.Dashboard.panelExplanation}
-                </Text>
+                <Separator className={classNames.separator} />
               </Stack.Item>
-              {!this.props.loadingExplanation[index] ? (
+              <Stack
+                tokens={{ childrenGap: "l2" }}
+                className={classNames.sectionIndent}
+              >
                 <Stack.Item>
-                  <Image
-                    src={`data:image/jpg;base64,${this.props.explanations.get(
-                      index
-                    )}`}
-                    width="700px"
-                    style={{ position: "relative", right: 85 }}
-                  />
+                  <Text variant="large" className={classNames.title}>
+                    {localization.InterpretVision.Dashboard.panelInformation}
+                  </Text>
                 </Stack.Item>
-              ) : (
-                <Stack.Item>
-                  <Spinner
-                    label={`${localization.InterpretVision.Dashboard.loading} ${item?.index}`}
+                <Stack.Item className={classNames.featureListContainer}>
+                  <List
+                    items={this.state.metadata}
+                    onRenderCell={onRenderCell}
                   />
                 </Stack.Item>
-              )}
+              </Stack>
             </Stack>
-            <Stack.Item>
-              <Separator className={classNames.separator} />
-            </Stack.Item>
-            <Stack
-              tokens={{ childrenGap: "l2" }}
-              className={classNames.sectionIndent}
-            >
+            <Stack>
+              <Stack.Item>
+                <Separator className={classNames.separator} />
+              </Stack.Item>
               <Stack.Item>
                 <Text variant="large" className={classNames.title}>
-                  {localization.InterpretVision.Dashboard.panelInformation}
+                  {localization.InterpretVision.Dashboard.panelExplanation}
                 </Text>
               </Stack.Item>
-              <Stack.Item className={classNames.featureListContainer}>
-                <List
-                  items={this.state.metadata}
-                  onRenderCell={this.onRenderCell}
-                />
-              </Stack.Item>
+              <Stack>
+                {
+                  <ComboBox
+                    id="objectSelection"
+                    label={"Choose a detected object"}
```

Comment:
move text to localization

Reply:
Done for English -- but do I need to update all of the languages? Or is this auto-generated?